doc/reshaping.rst

```diff
@@ -134,6 +134,36 @@ pandas, it does not automatically drop missing values. Compare:
 We departed from pandas's behavior here because predictable shapes for new
 array dimensions is necessary for :ref:`dask`.
 
+Stacking different physical variables together
```

Comment:
nit: let's remove the word "physical" here.

Reply:
done